plenum/server/batch_handlers/three_pc_batch.py

```diff
@@ -41,9 +43,12 @@ def from_pre_prepare(pre_prepare, state_root, txn_root, primaries, valid_digests
                             txn_root=txn_root,
                             primaries=primaries,
                             valid_digests=valid_digests,
+                            digest=digest,
```

Comment:
I think `pre_prepare.digest` can be used instead of explicit passing

Reply:
ok